README.md

```diff
@@ -58,4 +58,16 @@ Please refer to [official documentation for Earthly Satellites](https://docs.ear
 If you are a member of ExpressVPN, you can get access to the same Earthly organization used in our CI. The organization is named `expressvpn`, inside which contains a satellite named `wolfssl-rs`.
 
 If you are not a member of ExpressVPN, you may set up your own Earthly satellite according the official instructions above.
- 
\ No newline at end of file
+
+# Releasing
+
+A GitHub Workflow is set up to automate the release of crates in this repo. Upon a release, it will create a release in GitHub and Crates.io
+
+To create a new release, follow the below steps:
+
+1. Bump the version in `<crate-name>/Cargo.toml`. We follow the semantic versioning pattern when deciding a new version number
+1. Open a PR, attach the `release` label to the PR
+1. Observe that a comment is add to the PR, indicating the current version and the upcoming version
+1. Merge the PR, a new version should be released to both GitHub and Crates.io
+
+The automation supports releasing multiple crates in one PR. It can be done by bumping the version in multiple `Cargo.toml` files.
```

Comment:
This might be tricky if/when wolfssl has a versioned (rather than path) dependency on wolfssl-sys? Depending what checks `cargo publish` does...

Reply:
Updated the release instruction that the two crates should be released together, but separately.